Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3841,6 +3841,26 @@ GstElement* MediaPlayerPrivateGStreamer::createVideoSink()
 {
     acceleratedRenderingStateChanged();
 
+    // Ensure the sink has the max-lateness property set.
+    auto exit = makeScopeExit([this]() {
```

Comment:
I think you can omit parenthesis and use &, `[&] {` maybe?

Reply:
I removed the () but I prefer to keep the this because I prefer to control the variables I capture when I write lambdas.